README.md

```diff
@@ -61,6 +61,8 @@ pip install -e .
 make install-dep
 make test  # just the unit tests
 tox  # all the code checks
+tox -e py39-pydocstyle # perform only pydocstyle tests (py39 is the version of the python interpreter you have installed)
+make diff_pydocstyle_report # run a diff to show how much changes where made in the docstryle
```

Comment:
Maybe mention `make help` to list the major Makefile targets?

Reply:
And s/docstryle/docstyle